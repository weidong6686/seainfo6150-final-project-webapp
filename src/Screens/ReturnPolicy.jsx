import React from 'react';
import './css/About.css';

const ReturnPolicy = (props) => {
    return (
        <div className="content">
            <h2 className="title">About Our Returns Policies</h2>
            <p>Shopee Mart offer returns for items within 30 days of receipt of shipment.</p>
            <p>For global return, it can take up to 25 days for an item to reach us once you return it. Once the item is received at our fulfillment center, it takes 2 business days for the refund to be processed and 3-5 business days for the refund amount to show up in your account.</p>
        </div>
    )
}

export default ReturnPolicy