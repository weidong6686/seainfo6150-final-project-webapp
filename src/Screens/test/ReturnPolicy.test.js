import React from "react";
import ReturnPolicy from "../ReturnPolicy.jsx";

describe("ReturnPolicy tests", () => {
  it("renders correctly", () => {
    const { container } = (<ReturnPolicy />);
    expect(container).toMatchSnapshot();
  });
});