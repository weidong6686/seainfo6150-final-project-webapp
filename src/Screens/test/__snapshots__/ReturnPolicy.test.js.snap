// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReturnPolicy tests renders correctly 1`] = `undefined`;
